### PR TITLE
Fix action to allow stripping prefix

### PR DESCRIPTION
**Pull request by jefflewis**

### CODE REVIEW

File: lib/fastlane/plugin/sentry/actions/sentry_upload_sourcemap.rb [hunk 1]
```diff
@@ -23,8 +23,8 @@ def self.run(params)
 
         command.push('--rewrite') if params[:rewrite]
         command.push('--no-rewrite') unless params[:rewrite]
-        command.push('--strip-prefix') if params[:strip_prefix]
-        command.push('--strip-common-prefix') if params[:strip_common_prefix]
+        command.push('--strip-prefix').push(params[:strip_prefix]) unless params[:strip_prefix].nil?
+        command.push('--strip-prefix-prefix').push(params[:strip_common_prefix]) unless params[:strip_common_prefix].nil?
```
Comment (jefflewis) [inline lib/fastlane/plugin/sentry/actions/sentry_upload_sourcemap.rb:27]:
```suggestion
        command.push('--strip-prefix').push(params[:strip_prefix]) unless params[:strip_prefix].nil?
        command.push('--strip-common-prefix').push(params[:strip_common_prefix]) unless params[:strip_common_prefix].nil?
```

**Review by bruno-garcia — COMMENTED** [review]
Could you please write a test to make sure this time the actual value is passed on?

**Review by bruno-garcia — APPROVED** [review]
Thanks @jefflewis

**Comment by bruno-garcia** [comment]
added missing changelog entries and disabled merge commit in the repo :|

https://github.com/getsentry/sentry-fastlane-plugin/blob/76c82b4fbc77698f5541a0a006f239d36f9cf143/CHANGELOG.md#unreleased

**Comment by bruno-garcia** [comment]
@jefflewis https://github.com/getsentry/sentry-fastlane-plugin/releases/tag/1.8.3 is out. Thanks for your PR

**Pull request merged**